rskj-core/src/main/java/co/rsk/remasc/RemascTransaction.java

```diff
@@ -53,7 +54,7 @@ public String toJsonString() {
     };
 
     public RemascTransaction(byte[] rawData) {
-        super(rawData);
+        super(RLP.decodeList(rawData));
```

Comment:
Maybe, out of scope, but it could be nice to have RLP logic decoupled from Transaction class, at least in this constructor

Reply:
@ajlopezrsk Now `RLP` logic is handled by the base class itself.